From f6fd0da29676bb60531c05745d23b815e6b44253 Mon Sep 17 00:00:00 2001
From: Magnus Edenhill <magnus@edenhill.se>
Date: Thu, 4 Jun 2015 12:12:23 +0200
Subject: [PATCH] mklove update: add --disable-silent-rules

---
 mklove/modules/configure.builtin | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mklove/modules/configure.builtin b/mklove/modules/configure.builtin
index 55d478d..308257f 100644
--- a/mklove/modules/configure.builtin
+++ b/mklove/modules/configure.builtin
@@ -46,4 +46,5 @@ mkl_toggle_option "Compatibility" "mk:MKL_MAINT_MODE" "--enable-maintainer-mode"
 mkl_option "Configure tool" "mk:PROGRAM_PREFIX" "--program-prefix=PFX" "Program prefix"
 
 mkl_option "Compatibility" "mk:DISABL_DEP_TRACK" "--disable-dependency-tracking" "Disable dependency tracking (no-op)"
+mkl_option "Compatibility" "mk:DISABL_SILENT_RULES" "--disable-silent-rules" "Verbose build output (no-op)"
 
-- 
2.1.4

